modules/installation-network-user-infra.adoc

```diff
@@ -44,7 +44,7 @@ All the {op-system-first} machines require network in `initramfs` during boot
 to fetch Ignition config from the machine config server.
 
 ifdef::ibm-z[]
-During the initial boot, the machines require an FTP server in order to
+During the initial boot, the machines require an HTTP or HTTPS server in order to
 establish a network connection to download their Ignition config files.
 
 Ensure that the machines have persistent IP addresses and host names.
```

Comment:
To perform the actual MPIO fail over check, leverage the bootlist command to update the boot device list with alternate logical device names.

The following command displays a boot list and it designates the possible boot devices for when the system is booted in normal mode.
$ bootlist -m normal -o
sda
To update the boot list for Normal boot mode, update additional alternate device names that you would like to add as follows:
$ bootlist -m normal -o /dev/sdc /dev/sdd /dev/sde
sdc
sdd
sde
Next node reboot will boot from the alternate device registered in the normal boot device list, when the previous/original boot disk path is down